src/platform/packages/shared/kbn-esql-composer/src/utils/formatters.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export function escapeColumn(column: string) {
```

Comment:
This is used in from so the name doesnt seem correct. Can we find something more generic?

Reply:
I don't think we need this at all, right? Doesn't the ast automatically wrap the fielnames in backticks when it's necessary?

Reply:
Agreed or the prettifier, I dont remember which exactly api is responsible for that